bin/v-add-mail-domain

```diff
@@ -157,44 +168,31 @@ if [[ "$MAIL_SYSTEM" =~ exim ]]; then
 
 	# Adding dkim support
 	if [ "$dkim" = 'yes' ]; then
-		cp -f $USER_DATA/mail/$domain.pem \
-			$HOMEDIR/$user/conf/mail/$domain/dkim.pem
+		cp -f "$USER_DATA/mail/$domain.pem" "$HOMEDIR/$user/conf/mail/$domain/dkim.pem"
 	fi
 
 	# Set permission
-	chmod 771 $HOMEDIR/$user/conf/mail/$domain
-	chmod 660 $HOMEDIR/$user/conf/mail/$domain/*
-	chmod 771 /etc/$MAIL_SYSTEM/domains/$domain_idn
-	chmod 770 $HOMEDIR/$user/mail/$domain_idn
+	chmod 771 "$HOMEDIR/$user/conf/mail/$domain"
+	chmod 660 "$HOMEDIR/$user/conf/mail/$domain/*"
```

Comment:
```suggestion
	chmod 660 "$HOMEDIR/$user/conf/mail/$domain"/*
```
Same here.

Reply:
That is for example such a case, within `"$over_all"` does not work...
The better solution is here assuming `chmod 660 "$HOMEDIR"/"$user"/conf/mail/"$domain"/*`, but it is very heavy redable...

Reply:
Yes, adding double quotes to each variable is the strictest way.

I just to show it `*` needs to be outside the double quotes.

Ultimately depends on the source reliability of variables.

Hestia's CLIs usually validates call parameters, so enclosing multiple variables in single double quotes can be considered safe and readable.

Reply:
Theoretically yes, call parameters are validated by HESTIA. But unfortunatelly it does not always work practically. I saw already a couple of cases, in which empty variable was "validated" as "passed" for example. I do not know, if it a bug or a feauture. And especially for such cases we use here these quotes.
Yes, it is not simple in generally. But good news: This error with `.*` inside of quotes was detected by hestia tests. One test especially with this script was failed.

Reply:
Indeed, Hestia still has a lot to improve...

But that really takes a lot of time and effort...

So its more feasible for us to add some protection by the way when we make changes.

Our goal is the same, to make Hestia better.

I should go to bed now, have a nice day, friend!